Tmain/list-kind-full.d/stdout-expected.txt

```diff
@@ -13,4 +13,7 @@ t	typedef	typedefs	on	referenceOnly:FALSE	nRoles:0
 u	union	union names	on	referenceOnly:FALSE	nRoles:0
 v	variable	variable definitions	on	referenceOnly:FALSE	nRoles:0
 x	externvar	external and forward variable declarations	off	referenceOnly:FALSE	nRoles:0
-L	label	goto label	off	referenceOnly:FALSE	nRoles:0
+L	label	goto labels	off	referenceOnly:FALSE	nRoles:0
+U	using	using statements	off	referenceOnly:FALSE	nRoles:0
+z	parameter	function parameters inside function definitions	off	referenceOnly:FALSE	nRoles:0
+A	NONE	make anonymous structure names depend on file name	off	referenceOnly:FALSE	nRoles:0
```

Comment:
Well.. this is used to switch on a particular behaviour on other tags.
The current ctags creates anonymous structure names that do not depend on the file name, they are just like __anonN where N is an increasing number which doesn't seem to be reset when the file changes. So say you have a source tree with several files. If you run ctags multiple times with different file sets the anonymous structure names will generally be different from run to run. This makes it hard to use ctags in "incremental" or "updating" mode.

The +A switch makes the structure names to be something like __anon HASH N where HASH depends on the file name and N is reset at the beginning of each file. This makes anonymous structure names more "stable".